server/tests/routes.test.js

```diff
@@ -57,6 +56,22 @@ describe("API POST Data", () => {
       res.body.every((store) => expectedStores.includes(store["store-id"]))
     ).toBe(true);
   });
+  it("should display stores in alphabetic order when without location", async () => {
+    const res = await request(app).post("/api/store/list").send({
+      queryLimit: 1,
```

Comment:
Increase the queryLimit to have more stores returned. 
Current test case would only verify whether the only store returned is the store with smallest name in alphabetic order.